### PR TITLE
Hide Post Content by Author with low reputation score by default

### LINKED ISSUE #471: Hide by default comment/post from author with less than 0 reputation
Reported by @the-ego-is-you:
> There's a nsfw issue, very obvious in the comment section on this post
https://busy.org/facebook/@doitvoluntarily/facebook-denies-helping-advertisers-target-vulnerable-teen-users#@doitvoluntarily/re-the-ego-is-you-re-doitvoluntarily-facebook-denies-helping-advertisers-target-vulnerable-teen-users-20170502t101947307z

Last comment should not be visible by default. We can rely on author reputation to hide comment.

### PR DESCRIPTION
**Pull request by jm90m**
Fixes #471

Changes:
-  Add check for reputation score on Feed Story Component and Full Post body component

### CODE REVIEW

File: src/components/Comments/Comment.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ import Avatar from '../Avatar';
 import Body from '../Story/Body';
 import CommentFooter from '../CommentFooter/CommentFooter';
 import './Comment.less';
+import HiddenCommentMessage from './HiddenCommentMessage';
```
Comment (Sekhmet) [inline src/components/Comments/Comment.js]: Styles should be last import.

File: src/components/Comments/HiddenCommentMessage.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    />
+    <br />
+    <a role="presentation" onClick={onClick}>
+      <FormattedMessage id="reveal_comment" defaultMessage="Reveal comment" />
+    </a>
+  </div>
+);
+
+HiddenCommentMessage.propTypes = {
+  onClick: PropTypes.func.isRequired,
```
Comment (Sekhmet) [inline src/components/Comments/HiddenCommentMessage.js]: I would rather use non-required prop and add default as NOOP (`() => {}`).

File: src/post/HiddenPostMessage.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      <br />
+      <a role="presentation" onClick={onClick}>
+        <FormattedMessage id="display_post" defaultMessage="Display post" />
+      </a>
+    </h3>
+  </div>
+);
+
+HiddenPostMessage.propTypes = {
+  onClick: PropTypes.func.isRequired,
```
Comment (Sekhmet) [inline src/post/HiddenPostMessage.js]: Same here.

**Comment by jm90m** [comment]
@Sekhmet just updated based on your feedback

**Pull request merged**